src/engine/scenes/TestSceneKevin.cpp

```diff
@@ -79,10 +80,16 @@ namespace lei3d
 		physicsPlaygroundCollider->Init();
 		physicsPlaygroundCollider->SetColliderToModel(*playgroundModel);
 
+		// Test color source
```

Comment:
My only question is that I'm having a hard time seeing the color sphere in effect, where is it right now with reference to the scene and what am I supposed to see?

Reply:
Oh psych, I see it, the red in the back of the starting platform

Reply:
okay nvm I see